modular_chomp/code/modules/event/infectedroom.dm

```diff
@@ -90,8 +90,8 @@
 		message_admins("Infected room event started; Virus: <a href='?src=\ref[virus];[HrefToken()];info=1'>[virus.name()]</a>")
 
 /datum/event/infectedroom/announce()
-	command_announcement.Announce("Confirmed outbreak of level 5 biohazard aboard \the [location_name()]. \
-	All personnel must contain the outbreak.", "Biohazard Alert", new_sound = 'sound/AI/outbreak5.ogg')
+	command_announcement.Announce("Confirmed outbreak of level 7 biohazard aboard \the [location_name()]. All personnel must contain the outbreak.", "Infectious Contaminant", new_sound = 'sound/AI/outbreak7.ogg')
```

Comment:
I honestly am considering going back and changing this to instead be like, a janitorial issue, and make it not a level 7 alert, but a "hey janitor the air filters backfired pls go clean up" or whatever.

Right now it's level 7 because people _could_ be sick and it _could_ spread but this is not an ideal solution because right now it doesn't tell you where the infected room actually is.

Reply:
`Infectious Contaminant` to `Infectious Contaminant in [target_area.name]` would that make the name of the infected room show up?

Reply:
target_area might work, but yes.